internal/ent/schema/jobresult.go

```diff
@@ -0,0 +1,159 @@
+package schema
+
+import (
+	"time"
+
+	"entgo.io/contrib/entgql"
+	"entgo.io/ent"
+	"entgo.io/ent/schema"
+	"entgo.io/ent/schema/field"
+	"github.com/gertd/go-pluralize"
+
+	"github.com/theopenlane/core/internal/ent/generated/privacy"
+	"github.com/theopenlane/core/internal/ent/privacy/policy"
+	"github.com/theopenlane/core/pkg/enums"
+	"github.com/theopenlane/entx/history"
+)
+
+// JobResult holds the schema definition for the JobResult entity
+type JobResult struct {
+	SchemaFuncs
+
+	ent.Schema
+}
+
+// SchemaJobResultis the name of the schema in snake case
+const SchemaJobResult = "scheduled_job_result"
```

Comment:
I think this can just be `job_result` for consistency with the name of the schema + file?
```suggestion
const SchemaJobResult = "job_result"
```